OpenRA.Platforms.Default/FreeTypeFont.cs

```diff
@@ -0,0 +1,147 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2019 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using System;
+using System.Diagnostics.CodeAnalysis;
+using System.IO;
+using System.Runtime.InteropServices;
+using OpenRA.Primitives;
+
+namespace OpenRA.Platforms.Default
+{
+	[SuppressMessage("Microsoft.StyleCop.CSharp.NamingRules", "SA1307:AccessibleFieldsMustBeginWithUpperCaseLetter",
+		Justification = "C-style naming is kept for consistency with the underlying native API.")]
+	[SuppressMessage("Microsoft.StyleCop.CSharp.NamingRules", "SA1310:FieldNamesMustNotContainUnderscore",
+		Justification = "C-style naming is kept for consistency with the underlying native API.")]
+	internal static class FreeType
+	{
+		internal const uint OK = 0x00;
+		internal const int FT_LOAD_RENDER = 0x04;
+
+		internal static readonly int FaceRecGlyphOffset = IntPtr.Size == 8 ? 152 : 84; // offsetof(FT_FaceRec, glyph)
```

Comment:
These offsets were generated using a simple C program:
```c
#include <ft2build.h>
#include FT_FREETYPE_H

int main(int argc, char** argv)
{
    printf("FaceRecGlyphOffset %lu\n", offsetof(FT_FaceRec, glyph));
    printf("GlyphSlotMetricsOffset %lu\n", offsetof(FT_GlyphSlotRec, metrics));
    printf("GlyphSlotBitmapOffset %lu\n", offsetof(FT_GlyphSlotRec, bitmap));
    printf("GlyphSlotBitmapLeftOffset %lu\n", offsetof(FT_GlyphSlotRec, bitmap_left));
    printf("GlyphSlotBitmapTopOffset %lu\n", offsetof(FT_GlyphSlotRec, bitmap_top));
    printf("MetricsWidthOffset %lu\n", offsetof(FT_Glyph_Metrics, width));
    printf("MetricsHeightOffset %lu\n", offsetof(FT_Glyph_Metrics, height));
    printf("MetricsAdvanceOffset %lu\n", offsetof(FT_Glyph_Metrics, horiAdvance));
    printf("BitmapPitchOffset %lu\n", offsetof(FT_Bitmap, pitch));
    printf("BitmapBufferOffset %lu\n", offsetof(FT_Bitmap, buffer));
    
    return 0;
}
```

compiled and run as both 32 and 64 bit.